R/pkg/R/stats.R

```diff
@@ -52,22 +52,17 @@ setMethod("crosstab",
             collect(dataFrame(sct))
           })
 
-#' Calculate the sample covariance of two numerical columns of a SparkDataFrame.
-#'
 #' @param colName1 the name of the first column
 #' @param colName2 the name of the second column
-#' @return The covariance of the two columns.
```

Comment:
OK. I added this back. The doc should be very clear even without this return value. Indeed,  most functions do not document return value in SparkR. See what it looks like in the image attached in the next comment.